include/oneapi/dpl/pstl/omp/util.h

```diff
@@ -153,6 +155,87 @@ __process_chunk(const __chunk_metrics& __metrics, _Iterator __base, _Index __chu
     __f(__first, __last);
 }
 
+namespace __detail
+{
+
+template <typename _ValueType, typename... _Args>
+struct __enumerable_thread_local_storage
+{
```

Comment:
Even though these class templates are now internal, the API description for them is still a part of the backend API - you should know what can be done with the object obtained from the make function. We will therefore need to document it somewhere in sufficient detail - perhaps in the RFC for a start.

Reply:
Yes, I believe this interface is described in the RFC to some extent already.  I can make that more specific and explicit.